setup.py

```diff
@@ -14,6 +14,8 @@
         long_description = f.read()
 
     install_requirements = [
+        # TODO: Once released, we need to set regmod >= 0.0.8
+        # "regmod>=0.0.8"
```

Comment:
Thanks David! I will try to work on it this week.

Reply:
No rush as long as you can tolerate broken builds! By default we import regmod 0.0.7, which will fail if we use tobit or try to initialize without data